apps/web/src/components/tx-flow/flows/SafenetTokenTransfers/RecipientRow/index.tsx

```diff
@@ -0,0 +1,86 @@
+import AddressBookInput from '@/components/common/AddressBookInput'
+import TokenAmountInput from '@/components/common/TokenAmountInput'
+import DeleteIcon from '@/public/images/common/delete.svg'
+import { FormControl, Grid, IconButton, SvgIcon, Typography } from '@mui/material'
+import { useFormContext } from 'react-hook-form'
+import type { TokenTransfersParams } from '..'
+import { TokenTransfersFields } from '..'
+import { useTokenAmount, useVisibleTokens } from '../utils'
+
+export const RecipientRow = ({
+  index,
+  groupName,
+  removable = true,
+  remove,
+}: {
+  index: number
+  removable?: boolean
+  groupName: string
+  remove?: (index: number) => void
+}) => {
+  const balancesItems = useVisibleTokens()
+
+  const fieldName = `${groupName}.${index}`
+  const {
+    watch,
+    formState: { errors },
+  } = useFormContext<TokenTransfersParams>()
+
+  const recipient = watch(TokenTransfersFields.recipients)
+
+  // TODO: Review tokens available for selection and max amount
+  const selectedToken = balancesItems.find((item) => item.tokenInfo.symbol === 'USDC')
```

Comment:
I think we should use the addresses from the Safenet config instead of the name / symbol. There are many fake tokens that are called USDC for instance.